packages/build/src/plugins_core/functions/index.js

```diff
@@ -63,6 +68,22 @@ const getZisiParameters = ({ buildDir, featureFlags, functionsConfig, functionsD
   return { basePath: buildDir, config, manifest, featureFlags: zisiFeatureFlags }
 }
 
+// TODO: is this the best place for this logic? should this live elsewhere, e.g. in its own plugin?
+const writeToScheduleFile = async (dir, zisiResult) => {
```

Comment:
```suggestion
const writeToScheduleFile = async (buildDir, zisiResult) => {
```

Suggestion to rename `dir` to make it clear which directory is being used.